<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>5</y>
          <width>641</width>
          <height>311</height>
         </rect>
        </property>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>641</width>
           <height>311</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0" colspan="4">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="4">
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Films Matching Query</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="CDQuery"/>
          </item>
          <item row="7" column="0" colspan="4">
           <widget class="QLabel" name="L_MovieDetails">
            <property name="text">
             <string>Movie Details</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Film Name</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="4">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QListWidget" name="MatchedMovies">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="L_CDNumber">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>CD Number:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="CDNumber">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Number</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="1" rowspan="2" colspan="3">
           <widget class="QListWidget" name="MovieDetails">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>311</height>
         </rect>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>641</width>
           <height>311</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="5" column="2">
           <widget class="QLineEdit" name="TB_MovieLang"/>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="TB_VidFormat"/>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="TB_MovieVideoQuality"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Catagory</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="4">
           <widget class="QPushButton" name="AddCD">
            <property name="text">
             <string>Add CD</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Video Format</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="TB_MovieName"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="TB_MovieYear"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Director</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Language</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Video Quality</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="TB_Category"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Year</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="TB_Director"/>
          </item>
          <item row="0" column="3" rowspan="8">
           <widget class="QListWidget" name="MovieListToAdd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="QPushButton" name="AddMovieToCD">
            <property name="text">
             <string>Add Movie to CD</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_CD_Record"/>
    <addaction name="actionRemove_CD_Record"/>
    <addaction name="actionFind_CD"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_CD_Record">
   <property name="text">
    <string>Add CD Record</string>
   </property>
  </action>
  <action name="actionRemove_CD_Record">
   <property name="text">
    <string>Remove CD Record</string>
   </property>
  </action>
  <action name="actionFind_CD">
   <property name="text">
    <string>Find CD</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
